biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -472,13 +486,29 @@ private void getDependencies(Map<Program, Dependency> deps, EnumSet<MavenScope>
 			}
 		}
 
+		if (useDependencyManagement) {
```

Comment:
What about BOMs which can appear in dependencyManagement sections? I guess it will just ignore `import` scope.

Reply:
Good point. I have to admit, I haven't known about the import scope till now. If transitive is true, it would make sense to include them as well. Otherwise we could also add a scope configuration, that lets one choose the desired scopes.

Reply:
I just went ahead and implemented it as I suggested. `transtive=true` and `depManagment=true` will now also consider the import scope.